packages/shared/src/server/auth/types.ts

```diff
@@ -16,6 +16,7 @@ const ApiKeyBaseSchema = z.object({
   orgId: z.string(),
   plan: z.enum(plans as unknown as [string, ...string[]]),
   rateLimitOverrides: CloudConfigRateLimit.nullish(),
+  isIngestionSuspended: z.boolean(),
```

Comment:
As said in a previous comment, I'd decouple this from auth and make it a separate cache + condition check.